Add Wasabi glossary (#363)
zkSNACKs/WasabiDoc: docs/glossary/README.md

---
{
  "title": "Glossary",
  "description": "Explanations of common words used in Wasabi and regarding Bitcoin Privacy with links to the docs for more details. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
}
---

# Glossary

This glossary contains the explanations of common words used in Wasabi and regarding Bitcoin Privacy in general.

> Every voice contains a link to the respective full document for more details.

Reviewer: `Every voice` What do you mean here, I'm not familiar with that expression...

Author: > `Every voice` What do you mean here, I'm not familiar with that expression...

Sorry, meant "Every item".

## Bitcoin in general

:::details
### BIP

> Bitcoin Improvement Proposals.

Reviewer 2: ```suggestion
Bitcoin Improvement Proposal.
```

Isn't singular better since BIPs is the plural one?

Author: > Following [315](https://github.com/zkSNACKs/WasabiDoc/pull/315), this could be a good place to insert the "two weeks" explanation.

Omg you are right 😂

> A set of proposals that members of the bitcoin community have submitted to improve bitcoin.

Reviewer 2: ```suggestion
A proposal that members of the bitcoin community have submitted to improve bitcoin.
```
If we use the singular version.

Read more: [Wasabi Supported BIPs](/using-wasabi/BIPs.md)
:::

:::details
### Cold Storage

> Refers to keeping a reserve of bitcoin offline.

Reviewer: ```suggestion
Refers to keeping a reserve of important Bitcoin secrets offline.
```

 bitcoin are not "offline"...

Cold storage is achieved when Bitcoin private keys are created and stored in a secure offline environment.
Cold storage is important for anyone with bitcoin holdings.
Online computers are vulnerable to hackers and should not be used to store a significant amount of bitcoin.
Read more: [Cold Wasabi Protocol](/using-wasabi/ColdWasabi.md)
:::

:::details
### Fees

> The sender of a transaction often includes a fee to the network for processing the requested transaction.

Reviewer: ```suggestion
The sender of a transaction includes a fee to the network for processing the requested transaction.
```

I think the days of 0 fee are over, so no need to mention it here.

> Most transactions require a minimum fee of 0.5 mBTC.

Reviewer 2: Why 0.5 mBTC?
Also, couldn't mBTC be confusing? I don't think it's used often in the documentation (maybe not used at all?).

> Most transactions require a minimum fee of 0.5 mBTC.

Reviewer: ```suggestion
```

I don't think we use `mBTC` anywhere else in the doc.
Also, there is no "minimum" fee, as miners can include transactions without a fee.

Read more: [What fee should I select?](/FAQ/FAQ-UseWasabi.md#what-fee-should-i-select)
:::

:::details
### Hardware Wallet

> A hardware wallet is a special type of bitcoin wallet which stores the user’s private keys in a secure hardware device.

Reviewer: ```suggestion
A hardware wallet is a special type of bitcoin wallet which generates stores the user’s private keys in a tailor made hardware device.
```

hardware wallet are not necessarily secure... it's just dedicated hardware.

Read more: [Hardware Wallet FAQs](/FAQ/FAQ-UseWasabi.md#hardware-wallet)
:::

:::details
### Lightning Network

Lightning Network is a proposed implementation of Hashed Timelock Contracts (HTLCs) with bi-directional payment channels which allows payments to be securely routed across multiple peer-to-peer payment channels.
This allows the formation of a network where any peer on the network can pay any other peer even if they don’t directly have a channel open between each other.
Read more: [Use Lightning](/why-wasabi/10Commandments.md#_10-use-lightning)
:::

:::details
### Mempool

> The bitcoin Mempool (memory pool) is a collection of all transaction data in a block that have been verified by bitcoin nodes, but are not yet confirmed.

Reviewer: ```suggestion
The bitcoin mempool (memory pool) is a collection of all transaction data in a block that have been verified by a bitcoin node, but are not yet confirmed.
Contrarily to the blockchain, there is no consensus of the mempool and every full node has it's own unique one.
```

Read more: [How does Wasabi know of incoming transactions to the mempool?](/FAQ/FAQ-UseWasabi.html#how-does-wasabi-know-of-incoming-transactions-to-the-mempool)
:::

:::details
### Multisignature

> Multisignature (multisig) refers to requiring more than one key to authorize a bitcoin transaction.

Reviewer: ```suggestion
Multisignature (m-of-n multisig) refers to requiring more than one key to authorize a bitcoin transaction.
Only with `m` signatures of `n` public keys can the bitcoin be spend.
```

Read more: [Can I generate a multi signature script?](/FAQ/FAQ-UseWasabi.md#can-i-generate-a-multi-signature-script)
:::

:::details
### Output

> Output, transaction output, or TxOut is an output in a transaction which contains two fields: a value field for transferring zero or more satoshis and a pubkey script for indicating what conditions must be fulfilled for those satoshis to be further spent.

Reviewer: Can the txOut be 0 sats?

Reviewer 3: https://bitcoin.org/en/glossary/output
Most of these explanations are from https://github.com/bitcoinbook/bitcoinbook/blob/develop/glossary.asciidoc
So this must be correct.

:::

:::details

> ### P2WPKH

Reviewer: ```suggestion
### Pay-to-Witness-Public-Key-Hash (P2WPKH)
```

So to be consistent with UTXO or SPV below

> The signature of a P2WPKH (Pay-to-Witness-Public-Key-Hash) contains the same information as a P2PKH spending, but is located in the witness field instead of the scriptSig field.

Reviewer: ```suggestion
The signature of a P2WPKH contains the same information as a Pay-to-Public-Key-Hash (P2PKH) spending, but is located in the witness field instead of the scriptSig field.
```

The scriptPubKey is also modified.
Read more: [BIP 84 derivation scheme for P2WPKH based accounts](/using-wasabi/BIPs.md#bip-84-derivation-scheme-for-p2wpkh-based-accounts)
:::

:::details
### satoshi

A satoshi (sat) is the smallest denomination of bitcoin that can be recorded on the blockchain.
It is the equivalent of 0.00000001 bitcoin and is named after the creator of Bitcoin, Satoshi Nakamoto.
Read more: [How can I display the fee in satoshi per byte?](/FAQ/FAQ-UseWasabi.md#how-can-i-display-the-fee-in-satoshis-per-byte)
:::

:::details
### Simplified Payment Verification (SPV)

> SPV or simplified payment verification is a method for verifying particular transactions were included in a block without downloading the entire block.

Reviewer: ```suggestion
SPV is a method for verifying particular transactions were included in a block without downloading the entire block.
```

so to be consistent with P2WPKH above

The method is used by some lightweight Bitcoin clients.
Read more: [Wasabi Wallet under the hood](/building-wasabi/TechnicalOverview.md#wasabi-wallet-under-the-hood)
:::

:::details
### Unspent Transaction Output (UTXO)

UTXO is an unspent transaction output that can be spent as an input in a new transaction.
Read more: [How can I select UTXOs for CoinJoin?](/FAQ/FAQ-UseWasabi.md#how-can-i-select-utxos-for-coinjoin)
:::

## Privacy and Wasabi

:::details
### Address reuse

Address reuse is a serious privacy leak.

> If an output address has been reused it is very likely to be a payment output, not a change output.

Reviewer: ```suggestion
If an output address has been reused the same private key can spend now several coins, thus it is clear that all of these coins belong to one entity.
Further, it is very likely to be a payment output, not a change output.
```

This is because change addresses are created automatically by wallet software but payment addresses are manually sent between humans.
Read more: [Address reuse](/using-wasabi/AddressReuse.md)

:::details
### Anonimity Set (or anonset)

> The anonymity set is effectively the size of the group you are hiding in during a CoinJoin.

Reviewer: ```suggestion
The anonymity set is effectively the size of the group you are hiding in during a CoinJoin.
It's the quantity of equal value outputs of one CoinJoin transaction.
```

Read more: [What is the anonimity set?](/FAQ/FAQ-UseWasabi.md#what-is-the-anonymity-set)
:::

:::details
### Change address detection

Many bitcoin transactions have change outputs.
It would be a serious privacy leak if the change address can be somehow found, as it would link the ownership of the (now spent) inputs with a new output.
Read more: [Change coins](/using-wasabi/ChangeCoins.md)
:::

:::details
### CoinJoin

> CoinJoin is a trustless method for combining multiple Bitcoin payments from multiple spenders into a single transaction to make it more difficult for outside parties to determine which spender paid which recipient or recipients. 

Reviewer: ```suggestion
CoinJoin is a trustless method for combining multiple Bitcoin payments from multiple spenders into a single transaction to make it more difficult for outside parties to determine which spender paid which recipient.
```

Read more: [What is a CoinJoin?](/FAQ/FAQ-Introduction.md#what-is-a-coinjoin)
:::

:::details
### Coin Control

Coin control is a must learn if you care about your privacy in Bitcoin.
As can be understood from the name, this is a proper control of one's coins.
Read more: [Coin Control Best Practices](/FAQ/FAQ-UseWasabi.md#coin-control-best-practices)
:::

:::details
### Common-input-ownership heuristic

This is a heuristic or assumption which says that if a transaction has more than one input then all those inputs are owned by the same entity.
Read more: [Wasabi Wallet under the hood](/building-wasabi/TechnicalOverview.md#wasabi-wallet-under-the-hood)
:::

:::details
### Coordinator

The coordinator is zkSNACKs' server which creates CoinJoins and accepts UTXOs in the mix.
Read more: [Wasabi Wallet under the hood](/FAQ/FAQ-UseWasabi.md#how-does-my-wallet-communicate-with-the-wasabi-coordinator-server)
:::

:::details
### Dust

> Dust is a small portion of bitcoins which can lead to serious consequences about one's privacy.

Reviewer: What you are describing here is the forced address reuse attack.

Dust is a utxo that is uneconomical to spend [aka necessary fees are larger than sat amount]

Read more: [What is the dust threshold](/FAQ/FAQ-UseWasabi.html#what-is-the-dust-threshold)
:::

:::details
### KYC

> Know your customer (KYC) is the process of a business, identifying and verifying the identity of its clients.

Reviewer: ```suggestion
Know your customer (KYC) is the process of a business being forced to identify and verify the identity of its clients, and to share this information with a government.
```

knowing whom you deal with is not KYC - being forced to do so and sharing that information is.

}
---

# Glossary

This glossary contains the explanations of common words used in Wasabi and regarding Bitcoin privacy in general.
Every item contains a link to the respective full document for more details.

## Bitcoin in general

:::details
### BIP

Bitcoin Improvement Proposal.
A design document for introducing features or information to Bitcoin.
Read more: [Wasabi Supported BIPs](/using-wasabi/BIPs.md)
:::

:::details
### Cold Storage

Refers to keeping a reserve of important Bitcoin secrets offline.
Cold storage is achieved when Bitcoin private keys are created and stored in a secure offline environment.
Cold storage is important for anyone with bitcoin holdings.
Online computers are vulnerable to hackers and should not be used to store a significant amount of bitcoin.
Read more: [Cold Wasabi Protocol](/using-wasabi/ColdWasabi.md)
:::

:::details
### Fees

The sender of a transaction includes a fee to the network for processing the requested transaction.
Most transactions require a minimum fee.
Read more: [What fee should I select?](/FAQ/FAQ-UseWasabi.md#what-fee-should-i-select)
:::

:::details
### Hardware Wallet

> A hardware wallet is a special type of Bitcoin wallet which generates and stores the user’s private keys in a tailor made hardware device.

Reviewer 3: ```suggestion
A hardware wallet is a special type of Bitcoin wallet which generates and stores the user's private keys in a tailor made hardware device.
```

Read more: [Hardware Wallet FAQs](/FAQ/FAQ-UseWasabi.md#hardware-wallet)
:::

:::details
### Lightning Network

Lightning Network is a proposed implementation of Hashed Timelock Contracts (HTLCs) with bi-directional payment channels which allows payments to be securely routed across multiple peer-to-peer payment channels.

> This allows the formation of a network where any peer on the network can pay any other peer even if they don’t directly have a channel open between each other.

Reviewer 3: ```suggestion
This allows the formation of a network where any peer on the network can pay any other peer even if they don't directly have a channel open between each other.
```

Read more: [Use Lightning](/why-wasabi/10Commandments.md#_10-use-lightning)
:::

:::details
### Mempool

The Bitcoin Mempool (memory pool) is a collection of all transaction data in a block that have been verified by Bitcoin nodes, but are not yet confirmed.
Read more: [How does Wasabi know of incoming transactions to the mempool?](/FAQ/FAQ-UseWasabi.html#how-does-wasabi-know-of-incoming-transactions-to-the-mempool)
:::

:::details

> ### Multisignature

Reviewer 3: ```suggestion
### Multisignature (multisig)
```

> Multisignature (m-of-n multisig) refers to requiring more than one key to authorize a bitcoin transaction.

Reviewer 3: ```suggestion
Multisignature (m-of-n multisig) refers to requiring more than one key to authorize a Bitcoin transaction.
```

> Only with `m` signatures of `n` public keys can the bitcoin be spend.

Reviewer 3: ```suggestion
Only with `m` signatures of `n` public keys can the bitcoin be spent.
```

Read more: [Can I generate a multi signature script?](/FAQ/FAQ-UseWasabi.md#can-i-generate-a-multi-signature-script)
:::

:::details
### Output

Output, transaction output, or TxOut is an output in a transaction which contains two fields: a value field for transferring zero or more satoshis and a pubkey script for indicating what conditions must be fulfilled for those satoshis to be further spent.
:::

:::details
### Pay-to-Witness-Public-Key-Hash (P2WPKH)

The signature of a P2WPKH contains the same information as a Pay-to-Public-Key-Hash (P2PKH) spending, but is located in the witness field instead of the scriptSig field.
The scriptPubKey is also modified.
Read more: [BIP 84 derivation scheme for P2WPKH based accounts](/using-wasabi/BIPs.md#bip-84-derivation-scheme-for-p2wpkh-based-accounts)
:::

:::details

> ### satoshi

Reviewer 3: ```suggestion
### satoshi (sat)
```

> A satoshi (sat) is the smallest denomination of bitcoin that can be recorded on the blockchain.

Reviewer 3: ```suggestion
A satoshi is the smallest denomination of bitcoin that can be recorded on the blockchain.
```

It is the equivalent of 0.00000001 bitcoin and is named after the creator of Bitcoin, Satoshi Nakamoto.
Read more: [How can I display the fee in satoshi per byte?](/FAQ/FAQ-UseWasabi.md#how-can-i-display-the-fee-in-satoshis-per-byte)
:::

:::details
### Simplified Payment Verification (SPV)

SPV is a method for verifying particular transactions were included in a block without downloading the entire block.
The method is used by some lightweight Bitcoin clients.
Read more: [Wasabi Wallet under the hood](/building-wasabi/TechnicalOverview.md#wasabi-wallet-under-the-hood)
:::

:::details
### Unspent Transaction Output (UTXO)

UTXO is an unspent transaction output that can be spent as an input in a new transaction.
Read more: [How can I select UTXOs for CoinJoin?](/FAQ/FAQ-UseWasabi.md#how-can-i-select-utxos-for-coinjoin)
:::

## Privacy and Wasabi

:::details
### Address reuse

Address reuse refers to the use of the same address for multiple transactions, this is very bad for privacy.
Read more: [Address reuse](/using-wasabi/AddressReuse.md)

:::details

> ### Anonimity Set (or anonset)

Reviewer 3: ```suggestion
### Anonymity Set (anonset)
```

CoinJoin is a trustless method for combining multiple Bitcoin payments from multiple spenders into a single transaction to make it more difficult for outside parties to determine which spender paid which recipient.
Read more: [What is a CoinJoin?](/FAQ/FAQ-Introduction.md#what-is-a-coinjoin)
:::

:::details
### Coin Control

Coin control is a must learn if you care about your privacy in Bitcoin.
As can be understood from the name, this is a proper control of one's coins.
Read more: [Coin Control Best Practices](/FAQ/FAQ-UseWasabi.md#coin-control-best-practices)
:::

:::details
### Common-input-ownership heuristic

This is a heuristic or assumption which says that if a transaction has more than one input then all those inputs are owned by the same entity.
Read more: [Wasabi Wallet under the hood](/building-wasabi/TechnicalOverview.md#wasabi-wallet-under-the-hood)
:::

:::details
### Coordinator

The coordinator is a server which creates CoinJoins and accepts UTXOs in the mix.
Read more: [Wasabi Wallet under the hood](/FAQ/FAQ-UseWasabi.md#how-does-my-wallet-communicate-with-the-wasabi-coordinator-server)
:::

:::details
### Dust

Dust is an UTXO that is uneconomical to spend.

> Also, small portion of bitcoins can lead to serious consequences about one's privacy.

Reviewer 3: ```suggestion
Also, small portions of bitcoin can lead to serious consequences about one's privacy.
```

> An example would be the so called 'forced address reuse attack'.

Reviewer 3: ```suggestion
An example would be the so called `forced address reuse attack`.
```

Read more: [What is the dust threshold](/FAQ/FAQ-UseWasabi.html#what-is-the-dust-threshold)
:::

:::details
### KYC

Know your customer (KYC) is the process of a business being forced to identify and verify the identity of its clients, and to share this information with a government.
The term is also used to refer to the bank regulation which governs these activities.
Read more: [AML/KYC Information](/why-wasabi/TransactionSurveillanceCompanies.md#aml-kyc-information)
:::

:::details
### XPUB (or Extended Public Key)

An xpub, also know as Extended Public Key, is a part of BIP-32 that will allow you to observe your wallet without the private key (xpriv).
Read more: [Wasabi's Solution](/why-wasabi/BitcoinPrivacy.md#wasabi-s-solution-4)
:::

:::details
### #twoweeks

The #twoweeks is a fun inside joke often used in the Wasabi documentation and, more generally, in the Internet community.
In the case of Wasabi documentation, it usually indicates the arrival of a new function or update, to which the future date is still uncertain.

Eg. "Lightning Network is coming to Wasabi in #twoweeks"
:::

:::details
### Wallet fingerprinting

> A careful analyst sometimes deduce which software created a certain transaction, because the many different wallet softwares don't always create transactions in exactly the same way.

Reviewer 3: ```suggestion
A careful analyst sometimes deduces which software created a certain transaction, because many different wallet softwares don't always create transactions in exactly the same way.
```

---
{
  "title": "Glossary",
  "description": "Explanations of common words used in Wasabi and regarding Bitcoin Privacy with links to the docs for more details. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
}
---

# Glossary

This glossary contains the explanations of common words used in Wasabi and regarding Bitcoin Privacy in general.
Every item contains a link to the respective full document for more details.

## Bitcoin in general

:::details
### BIP

Bitcoin Improvement Proposal.

> A proposal that members of the bitcoin community have submitted to improve bitcoin.

Reviewer 2: ```suggestion
A proposal that members of the Bitcoin community have submitted to improve Bitcoin.
```

Read more: [Wasabi Supported BIPs](/using-wasabi/BIPs.md)
:::

:::details
### Cold Storage

Refers to keeping a reserve of important Bitcoin secrets offline.

> Cold storage is achieved when Bitcoin private keys are created and stored in a secure offline environment.

Reviewer 4: I think that cold storage refers just to storing and not also to the creation of private keys.

Reviewer 2: I think it is correct to say both, if your keys were online at some point, it is not an actual cold storage. 
Pages like [Bitcoin wiki](https://en.bitcoin.it/wiki/Cold_storage) confirm this idea (*without the private keys controlling them ever being online*) and imho most people refer to cold storage in this terms.

Reviewer 4: You are totally right, of course it is like this, sorry about that!

Cold storage is important for anyone with bitcoin holdings.
Online computers are vulnerable to hackers and should not be used to store a significant amount of bitcoin.
Read more: [Cold Wasabi Protocol](/using-wasabi/ColdWasabi.md)
:::

:::details
### Fees

The sender of a transaction includes a fee to the network for processing the requested transaction.
Most transactions require a minimum fee.
Read more: [What fee should I select?](/FAQ/FAQ-UseWasabi.md#what-fee-should-i-select)
:::

:::details
### Hardware Wallet

> A hardware wallet is a special type of bitcoin wallet which generates stores the user’s private keys in a tailor made hardware device.

Reviewer 2: ```suggestion
A hardware wallet is a special type of Bitcoin wallet which generates stores the user’s private keys in a tailor made hardware device.
```

> A hardware wallet is a special type of Bitcoin wallet which generates stores the user’s private keys in a tailor made hardware device.

Reviewer 4: ```suggestion
A hardware wallet is a special type of Bitcoin wallet which generates and stores the user’s private keys in a tailor made hardware device.
```

Read more: [Hardware Wallet FAQs](/FAQ/FAQ-UseWasabi.md#hardware-wallet)
:::

:::details
### Lightning Network

Lightning Network is a proposed implementation of Hashed Timelock Contracts (HTLCs) with bi-directional payment channels which allows payments to be securely routed across multiple peer-to-peer payment channels.
This allows the formation of a network where any peer on the network can pay any other peer even if they don’t directly have a channel open between each other.
Read more: [Use Lightning](/why-wasabi/10Commandments.md#_10-use-lightning)
:::

:::details
### Mempool

> The bitcoin Mempool (memory pool) is a collection of all transaction data in a block that have been verified by bitcoin nodes, but are not yet confirmed.

Reviewer 2: ```suggestion
The Bitcoin Mempool (memory pool) is a collection of all transaction data in a block that have been verified by Bitcoin nodes, but are not yet confirmed.
```

:::

:::details
### Unspent Transaction Output (UTXO)

UTXO is an unspent transaction output that can be spent as an input in a new transaction.
Read more: [How can I select UTXOs for CoinJoin?](/FAQ/FAQ-UseWasabi.md#how-can-i-select-utxos-for-coinjoin)
:::

## Privacy and Wasabi

:::details
### Address reuse

Address reuse is a serious privacy leak.
If an output address has been reused the same private key can spend now several coins, thus it is clear that all of these coins belong to one entity.
Further, it is very likely to be a payment output, not a change output.
This is because change addresses are created automatically by wallet software but payment addresses are manually sent between humans.
Read more: [Address reuse](/using-wasabi/AddressReuse.md)

:::details
### Anonimity Set (or anonset)

The anonymity set is effectively the size of the group you are hiding in during a CoinJoin.
It's the quantity of equal value outputs of one CoinJoin transaction.
Read more: [What is the anonimity set?](/FAQ/FAQ-UseWasabi.md#what-is-the-anonymity-set)
:::

:::details
### Change address detection

> Many bitcoin transactions have change outputs.

Reviewer 2: ```suggestion
Many Bitcoin transactions have change outputs.
```

:::

:::details
### CoinJoin

CoinJoin is a trustless method for combining multiple Bitcoin payments from multiple spenders into a single transaction to make it more difficult for outside parties to determine which spender paid which recipient.
Read more: [What is a CoinJoin?](/FAQ/FAQ-Introduction.md#what-is-a-coinjoin)
:::

:::details
### Coin Control

Coin control is a must learn if you care about your privacy in Bitcoin.
As can be understood from the name, this is a proper control of one's coins.
Read more: [Coin Control Best Practices](/FAQ/FAQ-UseWasabi.md#coin-control-best-practices)
:::

:::details
### Common-input-ownership heuristic

This is a heuristic or assumption which says that if a transaction has more than one input then all those inputs are owned by the same entity.
Read more: [Wasabi Wallet under the hood](/building-wasabi/TechnicalOverview.md#wasabi-wallet-under-the-hood)
:::

:::details
### Coordinator

The coordinator is zkSNACKs' server which creates CoinJoins and accepts UTXOs in the mix.
Read more: [Wasabi Wallet under the hood](/FAQ/FAQ-UseWasabi.md#how-does-my-wallet-communicate-with-the-wasabi-coordinator-server)
:::

:::details
### Dust

> Dust is a is an UTXO that is uneconomical to spend.

Reviewer 4: ```suggestion
Dust is an UTXO that is uneconomical to spend.
```

---
{
  "title": "Glossary",

>   "description": "Explanations of common words used in Wasabi and regarding Bitcoin Privacy with links to the docs for more details. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."

Reviewer 3: ```suggestion
  "description": "Explanations of common words used in Wasabi and regarding Bitcoin privacy with links to the docs for more details. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
```

}
---

# Glossary

> This glossary contains the explanations of common words used in Wasabi and regarding Bitcoin Privacy in general.

Reviewer 3: ```suggestion
This glossary contains the explanations of common words used in Wasabi and regarding Bitcoin privacy in general.
```

Every item contains a link to the respective full document for more details.

## Bitcoin in general

:::details
### BIP

Bitcoin Improvement Proposal.

> A proposal that members of the Bitcoin community have submitted to improve Bitcoin.

Reviewer 3: ```suggestion
A design document for introducing features or information to Bitcoin.
```

Read more: [Wasabi Wallet under the hood](/FAQ/FAQ-UseWasabi.md#how-does-my-wallet-communicate-with-the-wasabi-coordinator-server)
:::

:::details
### Dust

Dust is an UTXO that is uneconomical to spend.
Also, small portion of bitcoins can lead to serious consequences about one's privacy.
An example would be the so called 'forced address reuse attack'.
Read more: [What is the dust threshold](/FAQ/FAQ-UseWasabi.html#what-is-the-dust-threshold)
:::

:::details
### KYC

Know your customer (KYC) is the process of a business being forced to identify and verify the identity of its clients, and to share this information with a government.
The term is also used to refer to the bank regulation which governs these activities.
Read more: [AML/KYC Information](/why-wasabi/TransactionSurveillanceCompanies.md#aml-kyc-information)
:::

:::details
### XPUB (or Extended Public Key)

An xpub, also know as Extended Public Key, is a part of BIP-32 that will allow you to observe your wallet without the private key (xpriv).
Read more: [Wasabi's Solution](/why-wasabi/BitcoinPrivacy.md#wasabi-s-solution-4)
:::

:::details
### #twoweeks

> The #twoweeks is a fun inside joke often used in the Wasabu documentation and, more generally, in the Internet community.

Reviewer 3: ```suggestion
The #twoweeks is a fun inside joke often used in the Wasabi documentation and, more generally, in the Internet community.
```
